src/openrct2/game.h

```diff
@@ -17,6 +17,7 @@
 #ifndef _GAME_H_
 #define _GAME_H_
 
+#include "platform/platform.h"
```

Comment:
If that's the case then I'm just gonna go ahead and rewind this back and drop all the extra commits.